internal/utils/aggregator_test.go

```diff
@@ -178,3 +190,27 @@ func Test_check_if_all_required_columns_in_CSV(t *testing.T) {
 		t.Error("Expecting error to be returned as all required column not present")
 	}
 }
+
+func Test_Aggregate_data(t *testing.T) {
```

Comment:
Added a test on Aggregate to attest condition which fixed this bug!

Reply:
```suggestion
func TestAggregateDataNoRecords(t *testing.T) {
```

Reply:
```suggestion
func TestAggregateDataNoRecords(t *testing.T) {
```